generate.xml

```diff
@@ -1906,24 +1906,22 @@
       <package label="test"            name="boost_unit_test_framework-vc141" version="1.64.0.0" />
     </nuget>
     <nuget build_name="secp256k1" compiler="vc120" >
-      <package label="secp256k1"       name="secp256k1_vc120"                 version="0.1.0.14" />
+      <package label="secp256k1"       name="secp256k1_vc120"                 version="0.1.0.16" />
     </nuget>
     <nuget build_name="secp256k1" compiler="vc140" >
-      <package label="secp256k1"       name="secp256k1_vc140"                 version="0.1.0.14" />
+      <package label="secp256k1"       name="secp256k1_vc140"                 version="0.1.0.15" />
     </nuget>
     <nuget build_name="secp256k1" compiler="vc141" >
-      <package label="secp256k1"       name="secp256k1_vc141"                 version="0.1.0.14" />
+      <package label="secp256k1"       name="secp256k1_vc141"                 version="0.1.0.16" />
     </nuget>
     <nuget build_name="zmq" compiler="vc120">
-      <package label="zmq"             name="libzmq_vc120"                    version="4.2.20.1" />
+      <package label="zmq"             name="libzmq_vc120"                    version="4.2.30" />
     </nuget>
-    <!--
     <nuget build_name="zmq" compiler="vc140">
-      <package label="zmq"             name="libzmq_vc140"                    version="4.2.20.1" />
+      <package label="zmq"             name="libzmq_vc140"                    version="4.2.30" />
     </nuget>
     <nuget build_name="zmq" compiler="vc141">
-      <package label="zmq"             name="libzmq-vc141"                    version="4.2.2" />
+      <package label="zmq"             name="libzmq-vc141"                    version="4.2.30" />
```

Comment:
This is the wrong package, https://www.nuget.org/packages/libzmq-vc141/ vs https://www.nuget.org/packages/libzmq_vc141/